tests/flow/test_bulk_insertion.py

```
@@ -719,3 +716,96 @@ def test11_social_multiple_labels(self):
+        # create edge csv files
+        #-----------------------------------------------------------------------
+
+        edge_count = 0
+        for edge_csv in edge_csvs:
+            with open(edge_csv, 'w') as f:
+                writer = csv.writer(f)
+
+                # Header row
+                writer.writerow(['src', 'dest'])
+
+                count = random.randint(edge_rel_min, edge_rel_max)
```

Ruff (0.13.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)